newsfragments/3240.installation

```diff
@@ -0,0 +1 @@
+The SFTP frontend is now contingent on the "sftp" Tahoe-LAFS extra.  To use it, be sure to install at least "tahoe-lafs[sftp]".
```

Comment:
Should this have a corresponding documentation update somewhere? I don't know where the tutorial-style prose docs live.

Reply:
Thanks for that excellent point!  I've added something.